routes/stripe.js

```diff
@@ -8,17 +8,30 @@ var stripe = require('stripe')(stripeKeys.secretKey);
 
 module.exports = {
   single: function(transaction, callback) {
-    var charge = {
-      amount: transaction.amount,
-      currency: transaction.currency,
-      card: transaction.stripeToken
-    };
-    stripe.charges.create(charge, callback);
+    stripe.customers.create({
```

Comment:
For single donations we also store a customer.

This means we can capture their address and thus have better fraud control.